app/models/consolidated_health_answers.rb

```diff
@@ -0,0 +1,30 @@
+class ConsolidatedHealthAnswers
+  def initialize
+    @answers = {}
+  end
+
+  def add_answer(responder:, question:, answer:, notes: nil)
+    @answers[question] ||= []
+    @answers[question] << { responder:, answer:, notes: }
+  end
+
+  def to_h
```

Comment:
As @benilovj mentioned in person, some comments similar to those removed from `AppHealthQuestionsComponent` would help.

Reply:
Added in eb0be58f083225739a519d7bbdd3a4d67d09125d